zahlenRaten.py

```diff
@@ -0,0 +1,78 @@
+import random
+
+
+def compareDistance(distanceGuessToNumber, distanceLastGuessToNumber, number):
+
+    if distanceGuessToNumber > distanceLastGuessToNumber:
+        print("kälter")
+    elif distanceGuessToNumber < distanceLastGuessToNumber:
+        print("wärmer")
+
+
+def game():
+
+    number = random.randint(1, 100)
+    trys = 0
```

Comment:
```suggestion
    tries = 0
```

oder "versuche" oder "attempts"